application/helpers/expressions/em_manager_helper.php

```diff
@@ -9800,6 +9752,75 @@ public static function htmlSpecialCharsUserValue($string)
+        // GJ: I think this is to assure that starting values always have a seed entry and then be able to check more easily
+        // If there are other starting values, besides the seed.
+        // I think this piece of code may never be reached by the execution thread. There is always a seed.
+        if (!array_key_exists('seed', $_SESSION[$LEM->sessid]['startingValues'])){        
+            $_SESSION[$LEM->sessid]['startingValues']['seed'] = '';
+        }
+
+        // Check if there are other starting values, besides the seed
+        // NOTE: now that we use a seed, count($_SESSION[$LEM->sessid]['startingValues']) start at 1
+        if (count($_SESSION[$LEM->sessid]['startingValues']) > 1) {
+            foreach ($_SESSION[$LEM->sessid]['startingValues'] as $k => $value) {
+                if (isset($LEM->knownVars[$k])) {
+                    $knownVar = $LEM->knownVars[$k];
+                } elseif (isset($LEM->qcode2sgqa[$k])) {
+                    $knownVar = $LEM->knownVars[$LEM->qcode2sgqa[$k]];
+                } else {
+                    continue;
+                }
+                if (!isset($knownVar['jsName'])) {
+                    continue;
+                }
+                switch ($knownVar['type']) {
+                    case Question::QT_D_DATE: //DATE
+                        if (trim($value) == "") {
+                            $value = null;
+                        } else {
+                            // We don't really validate date here, anyone can send anything : forced too
+                            $dateformatdatat = getDateFormatData($LEM->surveyOptions['surveyls_dateformat']);
+                            $datetimeobj = new Date_Time_Converter($value, $dateformatdatat['phpdate']);
+                            $value = $datetimeobj->convert("Y-m-d H:i");
+                        }
+                        break;
+                    case Question::QT_N_NUMERICAL: //NUMERICAL QUESTION TYPE
+                    case Question::QT_K_MULTIPLE_NUMERICAL_QUESTION: //MULTIPLE NUMERICAL QUESTION
+                        if (trim($value) == "") {
+                            $value = null;
+                        } else {
+                            $value = sanitize_float($value);
+                        }
+                        break;
+                    case Question::QT_VERTICAL_FILE_UPLOAD: //File Upload
+                        $value = null;  // can't upload a file via GET
+                        break;
+                }
+                $_SESSION[$LEM->sessid][$knownVar['sgqa']] = $value;
+                $LEM->updatedValues[$knownVar['sgqa']] = array(
+                    'type' => $knownVar['type'],
+                    'value' => $value,
+                );
+                unset($_SESSION[$LEM->sessid]['startingValues'][$k]);
+            }
+            $LEM->_UpdateValuesInDatabase();
```

Comment:
Mybe unset `$_SESSION[$LEM->sessid]['startingValues']` at end ?

Reply:
This is the port of the PR already tested and honestly adding that line seems to terminative. 
Wouldn't be that terminative :)

I would leave it like this.